programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -336,10 +339,13 @@ static bool process_tunnel_commands(const tunnel_command *tnl_cmd, command_cb cb
                 break;
             }
             log_level lvl = log_level_value_of(tunnel_set_log_level_cmd.loglevel);
-            if (lvl != log_level_Unknown) {
+            // int lvl = get_debug_level(tunnel_set_log_level_cmd.loglevel); // can be used only after c-sdk PR is merged
+
+            if (lvl != -1) {
                 if (ziti_log_level() != lvl) {
-                    set_log_level(tunnel_set_log_level_cmd.loglevel);
+                    set_log_level(lvl);
                     ziti_log_set_level(lvl);
+                    ziti_tunnel_set_log_level(lvl);
```

Comment:
I think it's good to set the tunnel log level. I'm not sure that the tunnel log level and the ziti log level should be set to the same level though.

I'm not proposing that you go ahead and make this change, but perhaps we should have an additional parameter in the set log level command to specify which logger is affected? I'm curious to hear from @ekoby.